### PR TITLE
Remove unused Hallo.js plugins

### PR DESCRIPTION
**Pull request by willbarton**
This PR removes two Hallo.js plugins: the ask-tips Hallo plugin (which isn't being used anymore as of #4982 and #5024) and the edit-html (which was removed for users in #5055).

## Checklist

- [x] PR has an informative and human-readable title
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [x] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [x] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### CODE REVIEW

**Comment by Scotchester** [comment]
How about we keep the scope of this PR constrained to the final removal of the Tip and HTML plugins?

**Comment by willbarton** [comment]
@Scotchester should [this change have it's own PR](https://github.com/cfpb/cfgov-refresh/pull/5529/files#diff-dcf2a8703e945442ba8591d07085a645L492-R492)?

I'm thinking that as long as there's no change in functionality, we can keep the changes in one PR. So far this makes no user-facing changes.

**Comment by Scotchester** [comment]
> @Scotchester should [this change have it's own PR](https://github.com/cfpb/cfgov-refresh/pull/5529/files#diff-dcf2a8703e945442ba8591d07085a645L492-R492)?

That and I was assuming that that same piece, the table RTE, would need more changes to get fixed.
 
> I'm thinking that as long as there's no change in functionality, we can keep the changes in one PR. So far this makes no user-facing changes.

But I can get behind this principle.

**Comment by willbarton** [comment]
Yeah. Part of the reason why this is a draft. I'm not sure what it'll take :)

File: cfgov/v1/atomic_elements/organisms.py [hunk 1]
```diff
@@ -489,7 +489,7 @@ def __init__(self, table_options=None, attrs=None):
     def render(self, name, value, attrs=None):
         value = self.json_dict_apply(
             value,
-            partial(expand_db_html, for_editor=True)
+            expand_db_html
```
Comment (Scotchester) [inline cfgov/v1/atomic_elements/organisms.py]: You can revert this change; it'll be included in my PR for the `TableBlock` Hallo stuff.

```suggestion
            partial(expand_db_html, for_editor=True)
```

Comment (willbarton) [reply cfgov/v1/atomic_elements/organisms.py]: I dropped the commit that made that change.
